server/server.js

```diff
@@ -56,6 +56,18 @@ if (process.env.NODE_ENV === "production") {
 // 	res.sendFile(path.join(__dirname, '../client/dist/index.html'));
 // });
 
+app.use((err, req, res, next) => {
+  res.status(err.status || 500);
+  res.json({
+    error: true,
+    data: {
+      message: err.message,
+      stack: process.env.NODE_ENV !== "production" ? err.stack : undefined
+    }
+  });
+  next(err);
```

Comment:
I don't think you need a `next` here unless you have some additional handler following this. This seems like a logical stopping point for any unhandled requests.